tests/publiccloud/download_repos.pm

```diff
@@ -30,7 +30,7 @@ sub get_repo_status {
 sub run {
     my ($self, $args) = @_;
 
-    select_host_console();    # select console on the host, not the PC instance
+    $self->select_serial_terminal();
```

Comment:
Maybe we should think of using `serial_terminal()` always for `host_console()` ?! WDYT?

Reply:
AFAIK, the `root-console` and `user-console` are both "graphical" consoles. What I'm using is the serial terminal.

check https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/opensusebasetest.pm#L1190 it falls back to `root-console` but if `VIRTIO_CONSOLE` or some other faster console is available, it takes it.

So I agree, we would need to double check, which tests can be changed to something faster. But in general for text only tests I would recommend to use the `serial_terminal`.